### PR TITLE
[tests-only] Fix flaky 'create-space-from-resource'-e2e test

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
Fixes flaky e2e tests that were introduced with https://github.com/owncloud/web/pull/8730.

I'd prefer to wait for requests here, but there can be `n` requests, and we don't know `n` inside the test step.

Potentially fixes the broken pipeline in https://drone.owncloud.com/owncloud/ocis/21162/68/10.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

### CODE REVIEW

File: tests/e2e/support/objects/app-files/resource/actions.ts [hunk 1]
```diff
@@ -135,15 +136,14 @@ export const createSpaceFromSelection = async ({
   })
   await page.locator(util.format(resourceNameSelector, resources[0])).click({ button: 'right' })
 
-  await page.locator('text=Create Space from selection').first().click()
-  await page.locator('.oc-text-input').first().fill(spaceName)
+  await page.locator(createSpaceFromResourceAction).first().click()
+  await page.locator(resourceNameInput).first().fill(spaceName)
```
Comment (ScharfViktor) [inline tests/e2e/support/objects/app-files/resource/actions.ts:140]: do we need `first()` here?

File: tests/e2e/support/objects/app-files/resource/actions.ts [hunk 2]
```diff
@@ -135,15 +136,14 @@ export const createSpaceFromSelection = async ({
   })
   await page.locator(util.format(resourceNameSelector, resources[0])).click({ button: 'right' })
 
-  await page.locator('text=Create Space from selection').first().click()
-  await page.locator('.oc-text-input').first().fill(spaceName)
+  await page.locator(createSpaceFromResourceAction).first().click()
```
Comment (ScharfViktor) [inline tests/e2e/support/objects/app-files/resource/actions.ts:139]: same. without first()

File: tests/e2e/support/objects/app-files/resource/actions.ts [hunk 3]
```diff
@@ -135,15 +136,14 @@ export const createSpaceFromSelection = async ({
...
+  await page.locator(resourceNameInput).first().fill(spaceName)
   await page.locator(util.format(actionConfirmationButton, 'Create')).click()
   const response = await page.waitForResponse(
     (resp) =>
       resp.status() === 201 && resp.request().method() === 'POST' && resp.url().endsWith('/drives')
   )
```
Comment (ScharfViktor) [inline tests/e2e/support/objects/app-files/resource/actions.ts:145]:
```
const [response] = await Promise.all([
        page.waitForResponse((resp) => resp.status() === 201 && resp.request().method() === 'POST')&& resp.url().endsWith('/drives'),
        page.locator(util.format(actionConfirmationButton, 'Create')).click()
      ])
```

**Comment by JammingBen** [comment]
Converting this to a draft because we temporarily skip the tests to get the pipeline green for the deployment: https://github.com/owncloud/web/pull/8783. We can merge this afterwards to check, if it solves the issue.

**Review by kulmann — APPROVED** [review]

**Pull request merged**